HW_1/task_5.py

```diff
@@ -11,3 +11,12 @@
 11 5
 9.22
 """
+import math
+
+x1 = int(input("Введите координату x1: "))
+y1 = int(input("Введите координату y1: "))
+x2 = int(input("Введите координату x2: "))
+y2 = int(input("Введите координату y2: "))
+
+distance = float(round(math.sqrt((x2 - x1)**2 + (y2 - y1)**2), 2))
```

Comment:
Можно вполне обойтись без библиотеки `math`. И функция `float` тут также не нужна. Python сам разберется в какой тип данных переводить результат. Кроме того, не забывайте, пожалуйста, про PEP8.
```suggestion
distance = round(((x2 - x1) ** 2 + (y2 - y1) ** 2) ** 0.5, 2))
```